lib/backend.cc

```diff
@@ -224,7 +224,7 @@ bool ClProgram::setup(std::string &binary) {
 
   std::string name = Device.getInfo<CL_DEVICE_NAME>();
 
-  int build_failed = Program.compile("-x spir -cl-kernel-arg-info");
+  int build_failed = Program.build(symbolSupported() ? "-x spir -cl-kernel-arg-info -cl-take-global-address" : "-x spir -cl-kernel-arg-info");
```

Comment:
Line exceeds 80 column limit.

Reply:
Should be fixed, thanks for catching that.